CONTRIBUTING.md

```diff
@@ -24,8 +24,29 @@ Thanks for taking the time to contribute to SystemDS!
 The following are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
 
 ___
-
-## Code Style
+### Contribution Guidelines and Standards
+
+Before contributing a pull request for [review](https://github.com/apache/systemds/pulls),
+let's make sure the changes are consistent with the guidelines and coding style.
+
+#### General Guidelines and Philosophy for contribution
+
+*   Inclusion of unit tests when contributing new features, will help
+    1. prove that the code works correctly, and
+    2. guard against future breaking changes.
+*   Bug fixes may or may not require unit tests, because the presence of bugs usually
+    indicates insufficient test coverage.
```

Comment:
This second comment is incorrect, not because of the first part before comma but because of the second part.
I would like to remove this comment.